src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -135,7 +135,7 @@ public function process(ContainerBuilder $container)
                     return 0;
                 }
 
-                return ($a['position'] < $b['position']) ? -1 : 1;
+                return $a['position'] <=> $b['position'];
```

Comment:
Sorry, this is for the whole paragraph, not just that line, you can delete lines 134 to 137

Reply:
My bad, I made the change too quickly, without analyze the whole paragraph.
By the way, I've removed the useless code.